packages/claims/src/claimsIssuer/claimsIssuer.ts

```diff
@@ -33,17 +33,20 @@ export class ClaimsIssuer extends Claims implements IClaimsIssuer {
     if (typeof claim === 'string') {
       claim = this.jwt.decode(claim) as IPublicClaim;
     }
-    const { claimData, did } = claim;
-    const signedToken = await this.jwt.sign(
-      { claimData, did, signer: this.did },
-      {
-        algorithm: Algorithms.ES256,
-        issuer: this.did,
-        subject: claim.did,
-        noTimestamp: true,
-        expirationTimestamp,
-      }
-    );
+    const { claimData, did, credentialStatus } = claim;
+    const dataToSign = {
+      claimData,
+      did,
+      signer: this.did,
+      ...(!!credentialStatus && { credentialStatus }),
```

Comment:
Maybe this is a simple question I should know, but why the `!!` here? It seems to me like it would do the same thing without
```suggestion
      ...(credentialStatus && { credentialStatus }),
```